keyboards/mechwild/bbs/readme.md

```diff
@@ -0,0 +1,19 @@
+# BB Steno (BBS)
+
+![BBS](https://i.imgur.com/XIjlzKOh.png)
+
+A bare-bones stenography keyboard. No bells or whistles. Simple, cheap, effective, steno.
+
+* Keyboard Maintainer: [Kyle McCreery](https://github.com/kylemccreery)
+* Hardware Supported: BBS v0.1
+* Hardware Availability: [BBS on MechWild](https://mechwild.com/product/bb-steno/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```